app/code/Magento/CatalogInventory/Model/Stock/Item.php

```diff
@@ -401,7 +401,7 @@ public function getQtyIncrements()
                 if ($this->getUseConfigQtyIncrements()) {
                     $this->qtyIncrements = $this->stockConfiguration->getQtyIncrements($this->getStoreId());
                 } else {
-                    $this->qtyIncrements = (int) $this->getData(static::QTY_INCREMENTS);
+                    $this->qtyIncrements = (float) $this->getData(static::QTY_INCREMENTS);
```

Comment:
Are fractional Qty increments actually supported?

Reply:
>Are fractional Qty increments actually supported?

uhm, at configuration level I see usage of float value: 

https://github.com/magento/magento2/blob/068a97e8ae1615f580e492829bea688496ea4487/app/code/Magento/CatalogInventory/Model/Configuration.php#L278

but you are right, they don't seem to be allowed; the problem is that in the scope of MSI project we are considering them as float, maybe we should discuss a bit more about this